package.json

```diff
@@ -1,12 +1,13 @@
 {
-  "name": "hubot-stackstorm",
-  "description": "A hubot plugin for integrating with StackStorm event-driven infrastructure automation platform.",
-  "version": "0.4.5",
-  "author": "StackStorm, Inc. <info@stackstorm.com>",
+  "name": "hubot-stackstorm-auth",
```

Comment:
I have this automated to setup our stackstorm box and needed a different package name in npm. I can branch and change that back for the merge. Wasn't sure when this was going to get more attention.